### ISSUE TITLE
Add Virtual openmc input

### ISSUE THREAD

**Issue by guyshtot** [mccode-dev/McCode#1393]
Currently McStas supports event records from MCNP and Tripoli4, it would be nice to have support also for the open source code OpenMC.

**Comment by willend** [comment]
@guyshtot to my understanding @ebknudsen is / has been working on getting MCPL support into OpenMC, so perhaps this will actually be available pretty soon? 

https://github.com/mctools/mcpl/issues/56#issuecomment-1368251711

**Comment by ebknudsen** [comment]
Indeed! MCPL-support has been included in the OpenMC-tree, and will be available from the next release of OpenMC. If you can't wait, you may also try the development version of OpenMC - I use it on a daily basis so it is working quite well.
The relevant PR is here: https://github.com/openmc-dev/openmc/pull/2116
